app/controllers/application_controller.rb

```diff
@@ -4,6 +4,9 @@ class ApplicationController < ActionController::Base
   before_action :configure_permitted_devise_parameters, if: :devise_controller?
   before_action :set_locale
 
+  # before_filter :authenticate_user!, :set_current_user
```

Comment:
Was this line in there before?
Prefer to remove lines than comment - we still have history in git 😄